### ISSUE TITLE
Request for Keyboard Shortcut Functionality to Toggle Smooth Scrolling

### ISSUE THREAD

**Issue by denrinm** [Caldis/Mos#593]
**这个问题是否源自某个缺陷？如果是，可于此处描述一下 | Yes, it's a usability problem. Currently, I need to manually click on the "Smooth Scrolling" option within the MOS application to activate or deactivate it. This process is cumbersome, especially when I need to switch frequently.**

**说说你的新点子 | I would appreciate the addition of a feature that allows users to toggle the "Smooth Scrolling" option using a keyboard shortcut. This functionality would significantly enhance the user experience, as users would be able to easily and quickly activate or deactivate smooth scrolling as needed without having to navigate the menu.**

**还有别的替代方案吗？ | I attempted to create an AppleScript to control this option and assign it to a keyboard shortcut, but it appears that the "Smooth Scrolling" option does not respond to these AppleScript commands. A direct solution from within the application would be much more reliable and efficient.**

**额外说明 | Here is the checkbox for the "Smooth Scrolling" option within the application's interface:
![Screenshot 2023-05-18 at 14 39 04](https://github.com/Caldis/Mos/assets/122022063/920257f2-ed40-4194-adac-955af4eebd6a)

I hope that the addition of this feature will be considered, as it would greatly enhance the flexibility and user-friendliness of MOS. Thank you for your consideration.**

**Comment by octothorpe8** [comment]
> Wouldn't this option help if you need to change frequently?

Not really, because you have to hold down that block key continuously. I was just using a website (Miro.com) where smooth scrolling was causing the zoom to go completely crazy. I can use Control (the block key I set up), but that means I'm holding it down every single time I use the zoom controls. What I'd rather do is invoke a quick key shortcut to disable Mos during this work session, then re-enable it afterwards. This can obviously be accomplished by disabling Mos in its preferences, but that means opening a dialog and checking a box when a keyboard shortcut would be a lot quicker.
